### PR TITLE
post about merger

### PR DESCRIPTION
**Pull request by raynamharris**
cc @tracykteal 

see also https://github.com/swcarpentry/website/pull/806

### CODE REVIEW

File: _posts/2017-09-02-merger.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - blog
+comments: true
+show_meta: true
+authors: ["Rayna Harris", "Tracy Teal"]
+
+---
+
+*“If you want to go fast, go alone. If you want to go far, go together.”*
+
+Software Carpentry and Data Carpentry are currently independent organizations with their own fiscal sponsorship, Steering Committees, governance model, and bank accounts. However, as is perhaps no surprise that the organizations' operations have evolved to share memberships, infrastructure for workshop coordination, an instructor training program, and even some staff members. This 'separate but collaborative' organizational structure has allowed us to build a shared community of instructors with more than 1000 certified instructors and 47 current Member Organizations around the world.
```
Comment (tracykteal) [inline _posts/2017-09-02-merger.md]: no surprise that the -> no surprise the

File: _posts/2017-09-02-merger.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#### A very brief history of Software and Data Carpentry
+
+![History](/images/SWCDChistory.png "Milestones in the history of Software Carpentry and Data Carpentry")
+
+## What does this mean for our instructors?
+
+Software and Data Carpentry have already unified their instructor training to have one Carpentries instructor certification and program. Hence, the instructor training program will continue as it is. People who are already instructors can continue to teach workshops as they already do! Nothing substantive will change. There may be some updates on email list locations, but it will remain the joint Carpentries instructor community that it already is. 
+
+The new Carpentries Steering Committee will include elected positions, and instructors will be the electorate casting votes for those, as they have in the past for the Software Carpentry Steering Committee. More information about the elections will be coming out in October. Please also consider running yourself for a position on the Steering Committee to help guiding the Carpentries through this next phase! If you’re interested in more information on elections now, please contact Kate Hertweck (k8hertweck@gmail.com). 
+
```
Comment (tracykteal) [inline _posts/2017-09-02-merger.md]: ## What does this mean for our Member Organizations?

Memberships are already joint between Software and Data Carpentry, so there will be no changes with memberships. All signed, pending and upcoming membership agreements will remain valid and terms will continue to be the same with the Carpentries as the are with Software and Data Carpentry. We will be transitioning the Software Carpentry Advisory Council to a joint Carpentries council, likely in January, and will keep members updated on that transition.

**Comment by tracykteal** [comment]
I realized we should also include a 'What does this mean for our Member organizations?' section, so suggested text for that section.

**Comment by hlapp** [comment]
Great work!

**Comment by raynamharris** [comment]
Both changes are incorporated. Thanks!

**Pull request merged**